app/views/answers/_new_edit.html.erb

```diff
@@ -1,4 +1,4 @@
-<%# locals: { question, answer, readonly, locking } %>
+<%# locals: { template, question, answer, readonly, locking } %>
```

Comment:
this partial is used also for previewing a template, did you test if still works?